metadata-editor/app/controllers/EditsController.scala

```diff
@@ -193,6 +201,16 @@ object EditsController extends Controller with ArgoHelpers {
     edits
   }
 
+  def metadataFromUsageRights(usageRights: UsageRights): Option[ImageMetadata] = {
+    val toImageMetadata: PartialFunction[UsageRights, ImageMetadata] = (ur: UsageRights) => ur match {
+      case u: StaffPhotographer        => ImageMetadata(byline = Some(u.photographer), credit = Some(u.publication))
+      case u: ContractPhotographer     => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
+      case u: CommissionedPhotographer => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
```

Comment:
Can we double check (again..) with Jo that we always want the publication as credit (i.e. `... for the ...`) even for commissioned photographers? I know they don't for illustrators (for which we still need to add a category), which they credit as `Illustration: Joe Doh`, but maybe they do for photographers..?